amy/consents/admin.py

```diff
@@ -31,9 +87,13 @@ class ConsentAdmin(admin.ModelAdmin):
     def get_term_option_type(self, obj):
         return obj.term_option.option_type if obj.term_option else None
 
+    def has_delete_permission(self, *args, **kwargs):
```

Comment:
Not sure if you wanted this written explicitly or is this a mistake since the parent class defines the same method.

Reply:
Ths class does not inherit from the `ArchiveActionMixin` I'll add a comment here to make that more clear. I do not think admins should be able to archive individual user's consents.